keyboards/null/st110r2/readme.md

```diff
@@ -0,0 +1,31 @@
+# null/st110r2
+
+null ST110r2 is a custom PCB/keyboard project that supports multiple layouts and is built on top of the Filco and the clones alike.
+
+![st110r2_zoom](https://s3.amazonaws.com/null-src/images/posts/nkbm-st110r2/NKBM-ST110r21-both1.png)
+![st110r2_full](https://s3.amazonaws.com/null-src/images/posts/nkbm-st110r2/NKBM-ST110r21-both3.png)
+
+* Keyboard Maintainer: [user176176](https://github.com/user176176)
+* Hardware Supported: [null] 最高タイプ (SaikouType) NKBM-ST110r2.x keyboard PCBs with Teensy 2.0++.
+* Hardware Availability: [NKBM-ST110r2 - null-src.com](https://null-src.com/posts/nkbm-st110r2/post.php)
+
+Make example for this keyboard (after setting up your build environment):
+
+    qmk compile -kb null/st110r2 -km default
+
+Flashing example for this keyboard:
+
+    qmk flash -kb null/st110r2 -km default
```

Comment:
```suggestion
    make null/st110r2:default

Flashing example for this keyboard:

    make null/st110r2:default:flash
```

Reply:
From the previous PR I understood that it's not required. This is much clearer.